### DISCUSSION TITLE
Editing of CSS

### DISCUSSION THREAD

**Discussion by houek** [protz/thunderbird-conversations#2019] [Q&A]
Hi, is it possible to edit CSS of the addon manually or add custom.css?
I'm using dark mode with white message background and found annoying black border around whole conversation and message.
I'd like to remove that margin (no. 1 on picture) and set to white the background (no. 2).

![image](https://github.com/thunderbird-conversations/thunderbird-conversations/assets/16896953/cbb22980-d936-4a1e-b97f-15ec256a5dde)

Could anyone help?

**Comment by rickgordon** [comment]
I would also like to be able to modify or add css for the extension. Changes made in userChrome.css don't seem to be applied, even if set as `!important`.
